### DISCUSSION TITLE
Home Assistant Ip Statico

### DISCUSSION THREAD

**Discussion by homegiu** [indomus/forum#733] [HUB: Home Assistant]
Ciao a tutti. Ho installato HA OS su raspberry ed è collegato via wifi alla rete, lo raggiungo da fuori via duckdns. Ho bisogno di capire come assegnare ad HA un indirizzo ip interno statico (succede infatti che dopo un distacco della rete elettrica, quando si ricollegata, HA prende un indirizzo ip diverso).
Grazie

**Comment by snakuzzo** [comment]
Non impostare ip statico su HassOS.
Imposta l'assegnazione fissa dell'ip su base mac address direttamente sul router.
Tutti i router hanno questa funzionalità sotto le impostazioni LAN

**Reply by homegiu** [reply]
Grazie mille per la risposta. Ci ho messo un po', ma ho studiato come fare secondo le tue indicazioni e sembra funzionare.
